main/qml/Window.qml

```diff
@@ -159,20 +148,20 @@ ApplicationWindow {
         }
     }
 
-    property Tab currentTab: mainView.currentTab()
-    
+    property Tab currentTab: mainView.currentTab
+    property View currentView: mainView.currentView
 
     Item {
         anchors.fill: parent
         Keys.onPressed: {
-            var v = view(); if (v === undefined) return;
-            v.ctrl = (event.key == Qt.Key_Control) ? true : false;
+            var v = currentView; if (v === undefined) return;
+            if (event.key == Qt.Key_Control) v.ctrl = true;
             event.accepted = frontend.handleInput(event.text, event.key, event.modifiers)
             event.accepted = true;
```

Comment:
Yes, this is true. But it seems to me like we don't want Qt trying to handle keypresses. If we aren't handling it, shouldn't it do nothing, so the behavior can be common to all frontends?